src/Algebra/Definitions.agda

```diff
@@ -161,3 +161,15 @@ LeftDivides ∙ \\ = (LeftDividesˡ ∙ \\) × (LeftDividesʳ ∙ \\)
 
 RightDivides : Op₂ A → Op₂ A → Set _
 RightDivides ∙ // = (RightDividesˡ ∙ //) × (RightDividesʳ ∙ //)
+
+StarMakeRight : A → Op₂ A → Op₂ A → Op₁ A → Set _
+StarMakeRight e _+_ _∙_ _⁻* = ∀ x → (e + (x ∙ (x ⁻*))) ≈ (x ⁻*)
+
+StarMakeLeft : A → Op₂ A → Op₂ A → Op₁ A → Set _
+StarMakeLeft e _+_ _∙_ _⁻* = ∀ x →  (e + ((x ⁻*) ∙ x)) ≈ (x ⁻*)
+
+LeftInduction : Op₂ A → Op₂ A → Op₁ A → Set _
```

Comment:
Naming is quite challenging. Some papers also use `starDestructLeft`. We can go with `LeftFixedPoint`

Reply:
If you have reasons (like a standard paper on Kleene Algebra) for a name, you should leave a comment to that effect in the code. Something already used in the literature is definitely better than inventing something new (usually).

Reply:
So you recommend to revert last commit? I think we can keep the changes.

Reply:
Naming is hard. It might be a good idea to open an issue with suggested names, with references (that should also be in code comments) for people to discuss.

Reply:
For residuals (one-sided divisions), I use `／` and `＼`.